specs/serverspec-core.spec

```
@@ -16,28 +19,47 @@ Source0:          %{name}.tar.gz
 BuildArch:        x86_64
 BuildRoot:        %{_tmppath}/%{name}-%{version}-root
 
+%if 0%{?el7}
 BuildRequires:    %{scl}-ruby
 BuildRequires:    %{scl}-ruby-devel
 BuildRequires:    %{scl}-rubygem-rake
 BuildRequires:    %{scl}-rubygem-bundler
-BuildRequires:    git gcc-c++
 Requires:         %{scl}-rubygem-bundler
+%else
+BuildRequires: ruby
+BuildRequires: ruby-devel
+BuildRequires: rubygem-rake
+BuildRequires: rubygem-bundler
+Requires:      rubygem-bundler
+%endif
+BuildRequires: git gcc-c++ make
 
 %prep
 %setup -q -c
 
 %build
+%if 0%{?el7}
 source %{_scl_prefix}/enable
+%endif
 bundle install --standalone --binstubs --without=development
 
 %install
 rm -fr $RPM_BUILD_ROOT
 mkdir -p $RPM_BUILD_ROOT%{install_dir}
 cp -a * .bundle $RPM_BUILD_ROOT%{install_dir}
+%if 0%{?el7}
 cp -a /opt/rh/%{scl}/root/usr/share/gems/* $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/
+%else
+cp -a /usr/share/gems/* $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/
+sed -i s/'#!ruby -w'/'#!\/usr\/bin\/env ruby'/g $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/2.5.0/gems/diff-lcs-1.3/bin/*
+%endif
 install -d $RPM_BUILD_ROOT/usr/bin
 mv ./bin/serverspec $RPM_BUILD_ROOT/usr/bin/serverspec
+%if 0%{?el7}
 sed -i 's,@@SCL_PREFIX@@,%{_scl_prefix},g' $RPM_BUILD_ROOT/usr/bin/serverspec
+%else
+sed -i 's,@@SCL_PREFIX@@,,g' $RPM_BUILD_ROOT/usr/bin/serverspec
```

```suggestion
sed -i '/@@SCL_PREFIX@@/d' $RPM_BUILD_ROOT/usr/bin/serverspec
```

I believe it should be like this, but I did not test it :)